pyproject.toml

```diff
@@ -51,6 +51,7 @@ tests = [
     "sqlalchemy-stubs"  # Change when sqlalchemy is upgraded https://docs.sqlalchemy.org/en/14/orm/extensions/mypy.html
 ]
 google = [
+    "protobuf<=3.20.*",
```

Comment:
Can you add a comment similar to the one on L51 -- on why we have this pin

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/743/commits/be643b513292f6e51821103bc0aaa11ae12ec2ea)